web/src/Web.App/Web.App.csproj

```diff
@@ -36,4 +36,7 @@
         <PackageReference Include="SmartBreadcrumbs"/>
         <PackageReference Include="Swashbuckle.AspNetCore"/>
     </ItemGroup>
+    <ItemGroup>
+        <Folder Include="Pages\"/>
```

Comment:
for my understanding what is the Pages directory for?

Reply:
Rider must have added this for me. I'll remove. Good spot!